### ISSUE TITLE
Failed to read fixed header.

### ISSUE THREAD

**Issue by AyushBhat** [jgeboski/purple-facebook#39]
i get that error in every 5 minutes and it disconnects

**Comment by EionRobb** [comment]
Not much to go on there...
https://github.com/jgeboski/purple-facebook#debugging

**Comment by AyushBhat** [comment]
i really cant understand how to do that :(

**Comment by jgeboski** [comment]
Is this every time, or just some times?

**Comment by atondwal** [comment]
I had this bug too, but then I recompiled with --enable-debug and it went away :/

**Comment by jgeboski** [comment]
@atondwal unlikely it was due to enabling the debug build.  It was probably just a coincidence.

**Comment by turkishcoffee** [comment]
This happens to me as well... if I idle too long (usually several hours) I lose my connection.

Then the account is disabled, which I believe is how pidgin handles all accounts. Meaning I am unable to reconnect unless I manually click "re-enable"... but I think this is more pidgin and facebook's designs colliding in a counter-intuitive way.

Not that I wouldn't like to see it fixed (or more likely worked-around) I'm just saying I think it's less an issue with the plugin itself. :^)

**Comment by jgeboski** [comment]
As of the latest release, it should be automatically reconnecting after a timeout period.  Eventually, I will make it more streamlined.

**Comment by turkishcoffee** [comment]
Ah I seem to be a version behind, sorry!

**Comment by AyushBhat** [comment]
@atondwal  can you please tell me how did you re-build it?

**Comment by stehpancar** [comment]
Using the plugin: ID String: prpl-facebook Version: 0.1.0 (Downloaded 2015-06-30)
I got the "Failed to read fixed header." Error randomly once in a while, every couple of days during the last two weeks. Reactivating the Facebook account "repaired" it.

Since half an hour I'm on ID String: prpl-facebook Version: 4098e875ebcb
At the beginning the "Failed to read fixed header." error was shown to me, but I did not reactivate the Facebook account. Instead I enable the "Debug Log on Windows". In this log I could not find anything related to "fixed header". The Facebook plugin was loaded and _unloaded_ with all other account plugins.
I then reactivated the Facebook account in pidgin and the error went away. I now have restarted pidgin a dozen times with logging enabled and have so far not seen the "Failed to read fixed header." error again.

Let's see if it keeps popping up sporadically.
